llvm/test/CodeGen/SPIRV/debug-info/basic-global-di.ll

```diff
@@ -8,12 +8,12 @@
 ; CHECK-MIR-DAG: [[dwarf_version:%[0-9]+\:iid\(s32\)]] = OpConstantI [[type_i64]], 5
 ; CHECK-MIR-DAG: [[source_language:%[0-9]+\:iid\(s32\)]] = OpConstantI [[type_i64]], 3
 ; CHECK-MIR-DAG: [[debug_info_version:%[0-9]+\:iid\(s32\)]] = OpConstantI [[type_i64]], 21
-; CHECK-MIR-DAG: [[filename_str:%[0-9]+\:id\(s32\)]] = OpString 1094795567, 1094795585, 792805697, 1111638594, 1111638594, 1128481583, 1128481603, 1697596227, 1886216568, 1663985004, 0
+; CHECK-MIR-DAG: [[filename_str:%[0-9]+\:id\(s32\)]] = OpString 1094795567, 1094795585, 792805697, 1111638594, 1111638594, 1128481583, 1128481603, {{1697596227|1700545347}}, 1886216568, 1663985004, 0
```

Comment:
Is this the hash?

Reply:
Nope, much worse. This is ascii (or maybe utf-8) printed as decimal integers. `1697596227` is `0x652f4343`, and `1700545347` is `0x655c4343` - so the real diff is that we now accept `5c` ('\') in the second character as well as `2f` ('/').